Dockerfile

```
@@ -0,0 +1,3 @@
+FROM node:19
+
+WORKDIR /heisse-preise-de/
```

You could put the entrypoint in here and a default cmd as well
```suggestion
WORKDIR /heisse-preise-de/

EXPOSE 3000

LABEL org.opencontainers.image.source=https://github.com/yobst/heisse-preise-de
LABEL org.opencontainers.image.description="Yobst Price Scraper"

ENTRYPOINT npm
CMD ["run", "start"]
```

---

Changed it accordingly